give.php

```diff
@@ -5,7 +5,7 @@
  * Description: The most robust, flexible, and intuitive way to accept donations on WordPress.
  * Author: GiveWP
  * Author URI: https://givewp.com/
- * Version: 2.7.5
+ * Version: 2.8.0
```

Comment:
Please change this to 2.9.0

Reply:
fixed in https://github.com/impress-org/givewp/compare/dec997d3b90a...613679045944